pkg/jobcreator/run.go

```diff
@@ -46,20 +49,42 @@ func RunJob(
 	// wait a short period because we've just started the job creator service
 	time.Sleep(100 * time.Millisecond)
 
+	// Start run job trace
+	c, span := jobCreatorService.controller.tracer.Start(ctx.Ctx, "run_job",
```

Comment:
Note that this span shares the same name as the resource provider `run_job` span. A TraceQL query like `{name="run_job"}` will include spans from both the job creator and resource provider.

This behavior is considered acceptable because one can narrow their search by adding the service name. For example `{resource.service.name="job_creator" && name="run_job"}`.